src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/client/ErrorNotificationConfiguration.java

```diff
@@ -25,6 +28,16 @@ public BasicAuthRequestInterceptor basicAuthRequestInterceptor(
         return new BasicAuthRequestInterceptor(username, password, StandardCharsets.UTF_8);
     }
 
+    @Bean
+    public RequestInterceptor logRequestBodyInterceptor() {
```

Comment:
I don't think we can log request body, as there's a risk that it will contain sensitive information (OCR data).

We should log response body instead.

Reply:
response is logged #464 

furthermore, if such comment exists and thumbed up by original requestor - there's something missing in between

not minding the above, can this PR be closed as not relevant?

Reply:
We realised this when we having a discussion that ocr data also has pattern validation so there is a chance that if validation fails then ocr data will be logged. Hence the request

Reply:
> response is logged #464
👍